### PR TITLE
Tc 5241 users api v2 docs

### PR DESCRIPTION
**Pull request by olegtradecloud**
## Description

https://tradecloud.atlassian.net/browse/TC-5241

### Scope
This PR creates user api manual

### Submitter pre-flight check

- [x] Review your documentation
- [x] Add labels `needs reviewing`
- [x] Assign PR and JIRA ticket to 'Reviewer' (in 'Reviewing' state)

### Reviewer pre-flight check

- [ ] Review documentation in PR
- [ ] Remove label `needs reviewing`
- [ ] Merge the PR and remove the release for this branch from Gitbook
- [ ] Set Jira ticket to 'Released'

### CODE REVIEW

File: user/invite-new.md [hunk 1]
```diff
@@ -4,6 +4,32 @@ description: How to invite a new user
 
 # Invite user
 
-Explanation
+You can invite new user to your company:
```
Comment (roy-tc) [inline user/invite-new.md]: This should read `You can invite a new user to your company`.

File: user/update.md [hunk 2]
```diff
@@ -3,3 +3,36 @@ description: How to update your user's profile and settings.
 ---
 
 # Update user profile and settings
+
+User companyId and roles cannot be updated.
```
Comment (roy-tc) [inline user/update.md]: This is rather a note. Let's first describe where this route is used for, and after that add this note.

File: user/invite-new.md (hunk 1 above)
Comment (olegtradecloud) [reply user/invite-new.md]: Fixed, thanks

File: user/update.md (hunk 2 above)
Comment (olegtradecloud) [reply user/update.md]: Added description

File: user/invite-new.md [hunk 3]
```diff
@@ -4,6 +4,32 @@ description: How to invite a new user
...
 
-API
+## Request
+
+{% api-method method="post" host="https://api.accp.tradecloud1.com/v2" path="/user"%} 
+{% api-method-summary %} Create user {% endapi-method-summary %}
+{% api-method-spec %} 
+{% api-method-request %} 
+{% api-method-headers %} 
+{% api-method-parameter name="Authorization" type="string" required=true %} Authentication token {% endapi-method-parameter %}
```
Comment (marcmatt) [inline user/invite-new.md]: Value is: `Bearer <Authentication token>`
Change in all pages

File: user/list.md [hunk 4]
```diff
@@ -3,3 +3,34 @@ description: How to list your company's users
...
+{% api-method method="get" host="https://api.accp.tradecloud1.com/v2" path="/user-search/_search"%} 
+{% api-method-summary %} Search users {% endapi-method-summary %}
+{% api-method-spec %} 
+{% api-method-request %} 
+{% api-method-headers %} 
+{% api-method-parameter name="Authorization" type="string" required=true %} Authentication token {% endapi-method-parameter %} 
+{% endapi-method-headers %}
+{% api-method-query-parameters %} 
+{% api-method-parameter name="companyId" type="uuid" required=true %} Company Id {% endapi-method-parameter %}
+{% api-method-parameter name="query" type="string" required=false %} Query string {% endapi-method-parameter %}
```
Comment (marcmatt) [inline user/list.md]: Explain the query parameter, which fields can you search on

File: user/invite-new.md [hunk 5]
```diff
@@ -4,6 +4,32 @@ description: How to invite a new user
 
 # Invite user
 
-Explanation
+You can invite a new user to your company:
```
Comment (marcmatt) [inline user/invite-new.md]: You can invite a new user to your company.
An invitation email will be send to this user.

File: user/invite-new.md (hunk 3 above)
Comment (olegtradecloud) [reply user/invite-new.md]: Fixed

File: user/list.md (hunk 4 above)
Comment (olegtradecloud) [reply user/list.md]: Added description, please verify

File: user/invite-new.md (hunk 5 above)
Comment (olegtradecloud) [reply user/invite-new.md]: Fixed

**Pull request merged**